src/utilities/roundNumberToDecimalPlaces.ts

```diff
@@ -0,0 +1,5 @@
+// precise rounding https://www.jacklmoore.com/notes/rounding-in-javascript/
+export function roundNumberToDecimalPlaces(value: number, decimals: number) {
+  // @ts-ignore - string concatenation represents exponential number notation
+  return Number(Math.round(value + 'e' + decimals) + 'e-' + decimals);
+}
```

Comment:
On my continued exploration of floating point fun, I found that `toFixed(2)` (which uses `Math.round` for it's rounding) does not accurately round all floating point numbers.

Here's an interesting [blog post on rounding](https://medium.com/swlh/how-to-round-to-a-certain-number-of-decimal-places-in-javascript-ed74c471c1b8) which led me to [this solution](https://www.jacklmoore.com/notes/rounding-in-javascript/) which converts the numbers to exponential notation for more accurate rounding.